.travis.yml

```diff
@@ -26,17 +26,11 @@ install:
   # Useful for debugging any issues with conda
   - conda info -a
   # create environment and install dependencies
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION numpy scipy astropy pip matplotlib coverage pandas scikit-learn h5py six pytest-cov coveralls cached-property psutil
-  - source activate test-environment
-  - conda install -c conda-forge healpy aipy pycodestyle attrs h5py healpy pyyaml mpi4py
-  - pip install git+https://github.com/HERA-Team/pyuvdata.git
-  - pip install git+https://github.com/HERA-Team/pyuvsim.git
-  - pip install git+https://github.com/HERA-Team/linsolve.git
-  - pip install git+https://github.com/HERA-Team/hera_qm.git
-  - pip install git+https://github.com/HERA-Team/uvtools.git
-  - pip install git+https://github.com/HERA-Team/hera_sim.git
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION coverage pytest-cov coveralls
+  - conda activate test-environment
```

Comment:
It looks like this line is making the Travis builds die. I think it should be `source activate test-environment`.

Reply:
Yup, I've fixed that, thanks. It's now dying because of `pyuvsim`, which is one that I had forgotten to make these changes for. It should now have a PR, so we'll wait on that.